src/crystal/system/unix/login.cr

```diff
@@ -0,0 +1,11 @@
+require "c/unistd"
+
+module Crystal::System
+  def self.login
+    if login = LibC.getlogin
+      String.new(login)
+    else
+      nil
```

Comment:
It seems `else nil` is used only once in the entire repo: https://github.com/crystal-lang/crystal/blob/master/src/compiler/crystal/semantic/call_error.cr#L288 And this should probably be removed as well.